py.bzl

```diff
@@ -22,21 +22,6 @@ load(
 )
 
 def python_repos():
-    http_archive_with_pkg_path(
-        name = "ruamel_yaml",
```

Comment:
Was this unused?

Reply:
@cjwagner - yes, this is dead, superseded code. I overlooked it before but and removing now after investigation of the bazel build protocol and the fact that `ruamel.yaml` is declared in `requirements.txt` and installed via `pip`.